### DISCUSSION TITLE
MS Azure

### DISCUSSION THREAD

**Discussion by merlinbest5** [biigle/core#471] [Q&A] [answered]
Our organization is looking to migrate our video files from AWS to MS Azure, and we’re running into a playback issue when accessed through Biigle. The video displays and plays, but restarts at 00:00 when any action is taken. Any advice on what may be the issue would be greatly appreciated. No problem with the same file on AWS.

**Comment by mzur** [comment, marked as answer]
Could you please send me the ID of a video so I can try to reproduce this behavior?

**Reply by merlinbest5** [reply]
I have limited access at the moment but I believe this is one I was testing: https://biigle.de/videos/4982/annotations

**Reply by mzur** [reply]
Thanks! I was also talking with your colleagues via email and Rob Skelly was able to resolve the issue:

> The default API service version on accounts dated 2009. The Accept-Ranges header wasn't accepted until version 2011-08-18, and we needed that to accept Range headers properly (which contain a byte range to tell the server to send back the file from somewhere in the middle, instead of the beginning). Unfortunately the default service version can't be set except using a C# API, so I've copied one off the web and updated the account's API version to something recent.
>
> That, and making sure to add a MOOV atom to the video files using ffmpeg will make the videos work properly.

**Reply by andrewcuff** [reply]
We're having the same issue here at DFO NL region. I'm not sure I understand how to implement this fix based on the answer here. Can someone please provide me with instructions?

**Comment by merlinbest5** [comment]
We ran into this issue again, and enough time passed that I forgot about this fix. Sharing Rob's program for updating the API version in case it's helpful: https://gitlab.com/rskelly/azureblobdefaultversion
